lib/node_modules/@stdlib/utils/jsonify-keys/test/test.js

```diff
@@ -0,0 +1,171 @@
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var noop = require( '@stdlib/utils/noop' );
+var jsonifyKeys = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof jsonifyKeys, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function throws if not provided a string', function test( t ) {
+	var values;
+	var i;
+
+	values = [
+		3.14,
+		NaN,
+		true,
+		null,
+		void 0,
+		[],
+		{},
+		function noop() {}
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[i] ), TypeError, 'throws a type error when provided '+values[i] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			jsonifyKeys( value );
+		};
+	}
+});
+
+tape( 'the function throws if not provided a string (reviver)', function test( t ) {
+	var values;
+	var i;
+
+	values = [
+		3.14,
+		NaN,
+		true,
+		null,
+		void 0,
+		[],
+		{},
+		function noop() {}
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[i] ), TypeError, 'throws a type error when provided '+values[i] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			jsonifyKeys( value, noop );
+		};
+	}
+});
+
+tape( 'the function throws if provided a reviver argument which is not a function', function test( t ) {
+	var values;
+	var i;
+
+	values = [
+		'5',
+		3.14,
+		NaN,
+		true,
+		null,
+		void 0,
+		[],
+		{}
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[i] ), TypeError, 'throws a type error when provided '+values[i] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			jsonifyKeys( '{"a":"b"}', value );
+		};
+	}
+});
+
+tape( 'the function returns a JSON value if provided valid JSON', function test( t ) {
+	var expected;
+	var actual;
+
+	expected = {
+		'beep': 'boop'
+	};
+	actual = jsonifyKeys( '{beep:"boop"}' );
+	t.deepEqual( actual, expected, 'deep equal' );
+
+	expected = null;
+	actual = jsonifyKeys( 'null' );
+	t.equal( actual, expected, 'returns null' );
+
+	expected = true;
+	actual = jsonifyKeys( 'true' );
+	t.equal( actual, expected, 'returns true' );
+
+	expected = 3.14;
+	actual = jsonifyKeys( '3.14' );
+	t.equal( actual, expected, 'returns 3.14' );
+
+	t.end();
+});
+
+tape( 'the function returns an error if provided invalid JSON', function test( t ) {
+	var out = jsonifyKeys( '{beep:"boop}' );
```

Comment:
```suggestion
	var out = jsonifyKeys( '{ beep: "boop }' );
```
same comment on spacing for all other instances